.github/workflows/canary-integration-test.yml

```diff
@@ -781,3 +781,114 @@ jobs:
       if: failure()
       uses: mxschmitt/action-tmate@v3
       timeout-minutes: 120
+
+  rgw-multisite-testing:
+    runs-on: ubuntu-18.04
+    if: "!contains(github.event.pull_request.labels.*.name, 'skip-ci')"
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+
+    - name: install deps
+      run: |
+        tests/scripts/github-action-helper.sh install_deps
+        sudo apt-get install -y s3cmd
+
+    - name: setup minikube
+      uses: manusa/actions-setup-minikube@v2.3.1
+      with:
+        minikube version: 'v1.18.1'
+        kubernetes version: 'v1.19.2'
+        start args: --memory 6g --cpus=2
+        github token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: use local disk into two partitions
+      run: |
+        BLOCK=$(sudo lsblk --paths|awk '/14G/ {print $1}'| head -1)
+        BLOCK_DATA_PART=${BLOCK}1
+        sudo dmsetup version||true
+        sudo swapoff --all --verbose
+        sudo umount /mnt
+        # search for the device since it keeps changing between sda and sdb
+        sudo wipefs --all --force "$BLOCK_DATA_PART"
+        tests/scripts/create-bluestore-partitions.sh --disk "$BLOCK" --osd-count 2
+        sudo lsblk
+
+    - name: build rook
+      run: tests/scripts/github-action-helper.sh build_rook
+
+    - name: deploy first cluster rook
+      run: |
+        BLOCK=$(sudo lsblk|awk '/14G/ {print $1}'| head -1)
+        cd cluster/examples/kubernetes/ceph/
+        kubectl create -f crds.yaml -f common.yaml -f operator.yaml
+        yq w -i -d1 cluster-test.yaml spec.dashboard.enabled false
+        yq w -i -d1 cluster-test.yaml spec.storage.useAllDevices false
+        yq w -i -d1 cluster-test.yaml spec.storage.deviceFilter ${BLOCK}1
+        yq w -i -d1 cluster-test.yaml spec.cephVersion.image ceph/daemon-base:latest-pacific-devel
+        kubectl create -f cluster-test.yaml -f object-multisite.yaml -f toolbox.yaml
+        sleep 200
```

Comment:
put this  as function in the tests/scripts/github-action-helper.sh and also I see `multi-cluster-mirroring:` also has same commands so it will less duplicates